examples/browser/Dockerfile

```diff
@@ -29,10 +29,12 @@
 #    on the 3000 link. Please wait a few seconds before trying again as Theia
 #    may not have completely finished starting up yet.
 
-FROM node:7.10
+FROM node:8.6
```

Comment:
Have you considered using node "8" instead of the more specific 8.6? I guess there's a small risk that Theia would not work well with a more recent version eventually (8.7, 8.8, etc).

Reply:
I tested using the updated Dockerfile, and it seems to work fine. Thanks for fixing this!

Reply:
Right I'll change that to node 8 only